libcipher/utils.py

```diff
@@ -0,0 +1,61 @@
+import io
+import sys
+import string
+
+
+class EnglishChecker():
+    """This class serves the purpose of checking if a statement is
+       English or not. Two filter methods accomplish this by ensuring:
+        1. At least 85% of the message comprises of english alphabets
+        2. At least 25% of the words in the sentence are contained in
+           the dictionary file, an exhaustive list of english words
+    """
+
+    def __init__(self, filename):
+        self.filename = filename
+        self.LETTERS_AND_SPACE = string.ascii_letters + string.whitespace
+        content = filename.getvalue()
+        self.ENGLISH_WORDS = content.upper()
```

Comment:
Two issues here:

   1. `.getvalue()` is a StringIO method. You should be using `read()` or better yet `readlines()`.
   2. This keeps the whole dictionary as one big string. And later on we are searching through the string. This works, but is grossly inefficient. The dictionay will have one word per line. Read all the lines, convert them to upper case, and then make a it a `set`.